style.css

```diff
@@ -0,0 +1,12 @@
+div {
+    width: 500px;
+}
+
+#lyngkboard {
+  border: 2px solid black;
+  z-axis: -20;
+}
+
+#boardset {

```

Comment:
Is this necessary?

Reply:
It was for drag and drop images in checkers, i.e. the board was further away from the screen than the checker images.